runners/spark/src/test/java/org/apache/beam/runners/spark/SparkPortableExecutionTest.java

```diff
@@ -81,89 +76,6 @@ public static void tearDown() throws InterruptedException {
     sparkJobExecutor = null;
   }
 
-  @Test(timeout = 600_000)
-  public void testExecution() throws Exception {
```

Comment:
Just checking, are you removing this because more specific tests cover the general aspects of execution?

Reply:
Yes.

Reply:
This test is left over from before we had validates runner tests for the portable Spark runner.